### DISCUSSION TITLE
GitHub support is doing nothing about repeated harassment and bullying by certain members of a github community. I want to send a letter to the github legal department. Anyone have the address for such?

### DISCUSSION THREAD

**Discussion by myearwood1** [community/community#69047] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

GitHub support is doing nothing about repeated harassment and bullying by certain members of a github community. I want to send a letter to the github legal department. Anyone have the address for such?

**Comment by queenofcorgis** [comment]
Hi @myearwood1 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close and lock this post, but for this and any future incidents, please refer to those links.

Thank you!
